roles/edpm_nova/templates/02-nova-host-specific.conf.j2

```diff
@@ -2,8 +2,8 @@
 my_ip = {{ ctlplane_ip }}
 host = {{ canonical_hostname }}
 
-{% if edpm_nova_tls_certs_enabled | bool %}
 [libvirt]
-live_migration_with_native_tls = true
+live_migration_with_native_tls = {% if edpm_nova_live_migration_native_tls|bool %}true{% else %}false{% endif %}
```

Comment:
Can we not just do
live_migration_with_native_tls = {{edpm_nova_live_migration_native_tls |bool}} ?

Reply:
Will do that in a follow-up to re-enable parts of libvirt TLS